src/Controllers/API/GetEventsController.php

```diff
@@ -77,6 +77,12 @@ public function __invoke(Request $request, Response $response, array $args)
 
             if (count($data['data']) > 0) {
                 $data['message'] = '';
+                for ($i = 0; $i < count($data['data']); $i++) {
+                    if ($this->eventModel->hpIdsByEventId($data['data'][$i]['id'])) {
+                        $data['data'][$i]['people_attending'] = $this->eventModel
+                            ->hpIdsByEventId($data['data'][$i]['id'])[0]['people_attending'];
+                    }
```

Comment:
Product Owner requested a badge showing 0 attendees for events that can have attendees but don't.